servers/src/test/java/tachyon/worker/block/meta/StorageDirTest.java

```diff
@@ -43,7 +43,7 @@
   private TempBlockMeta mTempBlockMeta;
 
   @Rule
-  public ExpectedException thrown = ExpectedException.none();
+  private ExpectedException mThrown = ExpectedException.none();
```

Comment:
Sorry cc, I pushed a change to fix this wrong naming of thrown (which should be mThrown)---as it is blocking me from building the source. You need to merge that commit with your local changes now.